src/components/Counter.js

```diff
@@ -0,0 +1,28 @@
+import {useState} from "react";
+
+function Counter(){
+  let [num, setNum]= useState(0);
+  let incNum =()=>{
+    setNum(Number(num)+1);
```

Comment:
You do not have to cover it with Number. You already set it as number in line #4

Reply:
thank you, fixed in https://github.com/AlexeyAndrianov/cinemavision/pull/1/commits/72f66d1c93669bff738ba0df8dd38fbb962a8fc2